features/supplier_journey.feature

```diff
@@ -0,0 +1,23 @@
+@not-production
```

Comment:
And the same here - it would be nice for the supplier to log in and out, even if they don't have any services to edit. (They could maybe change their contact info once that feature is live?)